scenarios/04_Battle_of_Barnon.cfg

```diff
@@ -1611,14 +1971,15 @@
             side=1
             x,y=39,37 # Uri's original keep
         [/filter]
-        {VARIABLE gold_amt {ON_DIFFICULTY 60 50 40}}
+        {VARIABLE gold_amt {ON_DIFFICULTY4 60 50 40 30}}
```

Comment:
I wonder if nightmare difficulty will finally make those 3 be able to beat a good player. Guess we'll find out!

Reply:
I think I might actually have a few further things I want to do with them; see the FIXME at line 771, and the TODO at line 1962...

Reply:
(they can wait for a separate PR, though)